recipes/collada-dom/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "collada-dom" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rdiankov/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3be672407a7aef60b64ce4b39704b32816b0b28f61ebffd4fbd02c8012901e0d
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - boost
```

Do you require `Boost::Python`, otherwise use `boost-cpp`
```suggestion
    - boost-cpp
```